docs/immortal-empires/norsca.md

```diff
@@ -0,0 +1,30 @@
+# Norsca
+
+## Wulfrik the Wanderer - World Walkers
+
+* **Route I - Ragnarok**
+  * The End Times are nigh. Wulfrik has long cursed the gods for his predicament, but now realises he must pledge himself to one of them if he is to survive the coming apocalypse.
+  * Attain level 3 allegiance with a Chaos God.
+   * Defeat all of the Challengers sent against you.
+  * **Reward**:
+	* _Seafang Empowered_: Campaign movement range +30% (factionwide), enables teleportation for Wulfrik from port cities he owns (possible?)

```

Comment:
Does Wulfrik have any teleportation references in lore? If not I'd be wary of giving him that. Campaign movement range is an excellent idea for him though. 

Given that this objective relates to chaos gods I wonder if something with daemons/corruption/chaos as part of the reward would make sense?

Reply:
His ship the seafang goes into the realm of chaos and reappears at target location that's what the teleportation is about

Reply:
His ship works the same way spaceships in 40k work basically.

Reply:
In that case ignore my question! This is now just a mechanical question for @msolefonte

Reply:
We can give him that as a reward but it would be too big for a reward. I can see it being its own mod.

Reply:
Just gave it another thought and this is a no. Too much work for a reward.

Reply:
Just buff the Seafang spell then.

Reply:
@OhManTFE What should this be then? Remove the teleportation bit, leave the factionwide movement range bonus?

Reply:
We should always be looking opportunities to give players something really hype, within the bounds of scope, of course. I believe Morathi was a big success in that regard. The best types of rewards are ones that actually change how you play, giving more tools to the player mid-campaign and breaking the late game monotony.

I would like to propose something which I think should be relatively easy to manipulate (but I may be wrong, you tell me).

And that is that occupying/razing port settlements replenishes all your armies' movement range factionwide. Think of it like a mini version of Taurox's mechanic. It would get players thinking of all sorts of little combos to see how long they can keep it going.

And it will be all themed around the Seafang.

Otherwise, we just give factionwide +10 melee attack when fighting at sea... XD

Reply:
We could give a similar effect that Khornate factions have - +X% movement speed after winning a battle/razing a settlement, but doing it specifically after a port would be trickier. I don't think there's an existing group of "all settlements with ports" so we'd have to script something ourselves and that might even need to be provided a list of all settlements with ports. Alternatively, I _think_ we can get at the settlement after a battle and try and find out if it has a port building built/available as a shortcut but that would need some considerable digging.

Reply:
All right, well let me know if you find out whether it's feasible or not. I am happy to collect the list of port settlements if they are needed and we confirmed to be going ahead with this reward.

Reply:
"The Seafang sails upon the Winds of Chaos itself, transporting Wulfrik's armies vast distances in the blink of an eye."

Reply:
I like the campaign movement range after winning a port settlement battle thing. The problem is that I think it should be only movement in sea and not in land. But this is a good start. I will chekc it myself.